ledger-service/http-json/src/main/scala/com/digitalasset/http/HttpService.scala

```diff
@@ -312,9 +312,11 @@ object HttpService extends StrictLogging {
       ec: ExecutionContext,
       aesf: ExecutionSequencerFactory,
   ): Future[Error \/ DamlLedgerClient] =
-    LedgerClient(ledgerHost, ledgerPort, clientConfig, nonRepudiationConfig).map(
-      _.leftMap(Error.fromLedgerClientError)
-    )
+    LedgerClient
+      .fromRetried(ledgerHost, ledgerPort, clientConfig, nonRepudiationConfig, 600)
```

Comment:
Why is failing in 10 minutes better than failing in 30 minutes, or tomorrow, or 15 seconds...?

Reply:
10 minutes are following how long sandbox waits for postgres to be ready here: https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/scala/platform/store/dao/HikariJdbcConnectionProvider.scala#L90
I will similarly add another private val to the object.
However, do you think this should be something configurable in cli?